### PR TITLE
New package: exercism-3.0.10

### PR DESCRIPTION
**Pull request by SolitudeSF**

### CODE REVIEW

**Comment by SolitudeSF** [comment]
yikes, how the hell did i build it. now it doesnt wants to build even locally.

**Comment by xelxebar** [comment]
Oh wow. I *just* tried putting this together myself a couple hours ago... and ran into the same exact issue. Not sure how I got it working in the first place either. Maybe need to go through Exercism's Go course to figure out how go packaging works. heh.

**Comment by SolitudeSF** [comment]
its works if using `go get` but that shouldnt be necessary, i think.

File: srcpkgs/exercism/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'exercism'
+pkgname=exercism
+version=3.0.10
+revision=1
+wrksrc="cli-${version}"
+build_style=go
+go_import_path="github.com/exercism/cli/exercism"
```
Comment (maxice8) [inline srcpkgs/exercism/template]: this is wrong, the import path is the path to the github repository.

you meant 

```
go_import_path=github.com/exercism/cli
go_package="${go_import_path}/exercism"
```

Comment (SolitudeSF) [reply srcpkgs/exercism/template]: yeah, i misunderstood

**Pull request merged**